### PR TITLE
fix: import unimplemented query error on bsx

### LINKED ISSUE #3142: IdentityPopover component show 0
#### step to reproduce:
- go to /spotlight
- hover any Creator name
- 0 collected / 0 created / 0 sold

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3142

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
✅ IdentityPopover now works on bsx

but when I check my profile it say  
> "Last bought less than a minute ago"

File: queries/subsquid/general/userStatsByAccount.graphql [hunk 1]
```diff
@@ -19,6 +19,26 @@ query userStatsByAccount($account: String!) {
   ) {
     totalCount
   }
+  lastBought: nftEntitiesConnection(
```
[the remark this answers is not in the document]
Comment (vikiival) [reply queries/subsquid/general/userStatsByAccount.graphql:22]: Definitely

**Review by vikiival — APPROVED** [review]
LGTM if works

**Review by roiLeo — COMMENTED** [review]
It show "Last bought 4month ago" when I bought one 14 days ago

Capture d’écran 2022-06-16 à 10 08 29 AM
Capture d’écran 2022-06-16 à 10 08 47 AM

**Comment by Jarsen136** [comment]
hey, it is the same query as online. Pls compare it with 'kodadit.xyz'. 

It seems a new issue.👀

> It show "Last bought 4month ago" when I bought one 14 days ago
> 
> 
> 
> Capture d’écran 2022-06-16 à 10 08 29 AM
> 
> Capture d’écran 2022-06-16 à 10 08 47 AM
> 
> 
> 
>

**Comment by vikiival** [comment]
cc 7e4fccd200ccc185d0e59c68cd9c82e92a047e59

**Comment by yangwao** [comment]
pay 100 usd

**Comment by Jarsen136** [comment]
> pay 100 usd

Seems it does not work. Let me try again.

**Comment by Jarsen136** [comment]
pay 100

**Comment by prachi00** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 50.14 USD/KSM ~ 1.994 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xef72cab25893089e60d177087c18e246d186e5ad13041cdac20a628aa3afc0f4](https://kusama.subscan.io/extrinsic/0xef72cab25893089e60d177087c18e246d186e5ad13041cdac20a628aa3afc0f4)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)
